app/graphql/queries/ten_newest_reviews.rb

```diff
@@ -0,0 +1,11 @@
+module Queries
+  class TenNewestReviews < BaseQuery
```

Comment:
NB: Maybe rename this to ReviewsForHomepage to keep it consistent with what its being named on the FE?